src/mutations/UpdateProject/updateProject.js

```diff
@@ -0,0 +1,14 @@
+import {gql} from 'urql';
```

Comment:
`src/queries/UpdateProject/updateProject.js`
UpdateProject should be replaced with feature name